hibernate-core/src/test/java/org/hibernate/orm/test/query/criteria/CriteriaMultiselectGroupByAndOrderByTest.java

```diff
@@ -132,13 +133,15 @@ public void testSubqueryGroupBy(SessionFactoryScope scope) {
 	@Test
 	@Jira( "https://hibernate.atlassian.net/browse/HHH-17231" )
 	@SkipForDialect( dialectClass = SybaseASEDialect.class, reason = "Sybase doesn't support order by + offset in subqueries")
+	@SkipForDialect( dialectClass = AltibaseDialect.class, reason = "Altibase offset in limit clause starts from 1")
 	public void testSubqueryGroupByAndOrderBy(SessionFactoryScope scope) {
 		executeSubquery( scope, true, false );
 	}
 
 	@Test
 	@Jira( "https://hibernate.atlassian.net/browse/HHH-17231" )
 	@SkipForDialect( dialectClass = SybaseASEDialect.class, reason = "Sybase doesn't support order by + offset in subqueries")
+	@SkipForDialect( dialectClass = AltibaseDialect.class, reason = "Altibase offset in limit clause starts from 1")
```

Comment:
Is this still correct? It seems you adapted the `SqlAstTranslator` to respect this difference.

Reply:
Looks like usage like below is not affected by SqlAstTranslator(test source line 166)
Is there any way to change offset value like below in SqlAstTranslator?

subquery.orderBy(
		cb.desc( entityName )
).offset( 0 );

Reply:
See the comment I left in your translator.